doc/classes/AudioEffectReverb.xml

```diff
@@ -1,8 +1,11 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <class name="AudioEffectReverb" inherits="AudioEffect" category="Core" version="3.0.alpha.custom_build">
 	<brief_description>
+		Adds a Reverb audio effect to an Audio bus.
+		Simulate the sound of acoustic environments such as rooms, concert halls, caverns, or an open space.
```

Comment:
Simulate -> Simulates
"or open spaces." <- keep plural like all the other examples